src/components/Image/Image.jsx

```diff
@@ -97,8 +97,8 @@ const Image = ({
         const observer = new IntersectionObserver(
           (entries) => {
             entries.forEach((entry) => {
-              if (entry.isIntersecting && !actualSrcSet) {
-                srcSet && applySrcSet();
```

Comment:
A quanto ho capito srcSet può essere `undefined`, leggendo la funziona `getImageAttributes` che lo genera, quindi penso si possa rompere poi la funziona `applySrcSet` che chiama il metodo `filter` su `srcSet` senza verificare se è un array. Potresti provare inserendo per esempio un'immagine in formato svg, dovrebbe essere uno dei casi che si può rompere.

Reply:
@pnicolli ho fatto il test con un svg, tiff e un gif, funziona. Non saprei dirti se si romperà o no.

<img width="853" alt="Schermata 2023-01-24 alle 14 47 23" src="https://user-images.githubusercontent.com/60133113/214311612-ea58823e-80fd-4634-822a-ea7709031d89.png">